src/configuration/console-config.tsx

```diff
@@ -96,87 +95,81 @@ export const ConsoleConfigPage: FC<ConsoleConfigProps> = ({ brokerId }) => {
   }
 
   return (
-    <Stack key={'stack' + brokerId}>
-      <StackItem isFilled>
-        <Flex>
-          <FlexItem>
-            <FormGroup
-              label="Expose"
-              fieldId={'console-config-expose-formgroup'}
-              key={'formgroup-console-expose'}
-            >
-              <Checkbox
-                label="Expose Console"
-                isChecked={exposeConsole}
-                name={'check-console-expose'}
-                id={'check-expose-console'}
-                onChange={setConsoleExpose}
-              />
-            </FormGroup>
-          </FlexItem>
-          <FlexItem>
-            <FormGroup
-              label="ExposeMode"
-              fieldId={'console-config-exposemode-formgroup'}
-              key={'formgroup-console-exposemode'}
+    <Form isHorizontal isWidthLimited key={'form' + brokerId}>
+      <Grid hasGutter md={6}>
+        <FormFieldGroup>
+          <FormGroup
+            label="Expose"
+            fieldId={'console-config-expose-formgroup'}
+            key={'formgroup-console-expose'}
+          >
+            <Checkbox
+              label="Expose Console"
+              isChecked={exposeConsole}
+              name={'check-console-expose'}
+              id={'check-expose-console'}
+              onChange={setConsoleExpose}
+            />
```

Comment:
@lavocatt , The checkbox for "Expose Console" is pre-checked in the UI. However, it seems selecting or unselecting it doesn't affect functionality. Should we keep it, considering this observation? Also, should we add a mandatory symbol to clarify its importance?

Reply:
I don't understand, what do you mean by "pre-checked in the UI", the value isn't checked by default. Checking the value isn't mandatory either. Because the user can leave it unchecked if necessary.

Reply:
Yeah we leave it as checked by default. Otherwise our plugin cannot access the jolokia outside the openshift (in dev mode, in production the console/jolokia is accessed via the service)

Reply:
Ah it was in the console config, sorry for that guys, I was looking at the acceptor config..

Reply:
I'm adding the required field :slightly_smiling_face:

Reply:
Thanks!